src/xr_3da/entry_point.cpp

```diff
@@ -98,8 +100,7 @@ int main(int argc, char *argv[])
         }
 
         result = entry_point(commandLine);
-
-        free(commandLine);
+        if (commandLine) free(commandLine); {commandLine = NULL;};
```

Comment:
Inconsistent codestyle.
```suggestion
        if (commandLine)
            free(commandLine);
        commandLine = nullptr;
```
I wanted to say that it's prohibited to use standard functions and we should use `xr_free` instead of `free`. But as far as I know, xrMemory isn't initialized in this moment..